test/integration/targets/module_utils/module_utils/foo5.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python
+
+foo1 = "FOO FROM foo5.py"
```

Comment:
Since this is just renaming `test/integration/targets/module_utils/module_utils/foo.py` it can be omitted.